apps/camera/js/views/viewfinder.js

```diff
@@ -161,20 +161,18 @@ module.exports = View.extend({
     }, this.fadeTime);
   },
 
-  fadeIn: function(ms) {
+  fadeIn: function(first) {
```

Comment:
err.. nevermind, i think i misunderstood what this was indicating.. you could always use `firstRun` or something